var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -141,4 +163,19 @@ def install(self, spec, prefix):
         install_tree('lib', prefix.lib)
         install_tree('include', prefix.include)
         if '~mpi' in spec:
-            install('libseq/libmpiseq.a', prefix.lib)
+            lib_dsuffix = '.dylib' if sys.platform == 'darwin' else '.so'
+            lib_suffix = lib_dsuffix if '+shared' in spec else '.a'
+            install('libseq/libmpiseq%s' % lib_suffix, prefix.lib)
+
+        if '~mpi' in spec: # FIXME: A process or daemon was unable to complete a TCP connection to another process
+            with working_dir('examples', create=False):
+                mpirun = join_path(self.spec['mpi'].prefix.bin, 'mpirun') # FIXME: avoid hardcoding mpirun (could be mpiexec, mpiexec.hydra, etc)
```

Comment:
In fact on some mpi depending on the scheduler it could be completly something else than mpirun, mpiexec, mpirun.hydra. And the option passed to the executable will be different. For example `mvapich2+slurm` It would be `srun -n 2` 
Could the examples be activated by a variant ?